configuration.json

```diff
@@ -0,0 +1,24 @@
+{
+  "comments": {
+    "lineComment": "%%"
+  },
+  "brackets": [
+      ["{", "}"],
+      ["[", "]"],
+      ["(", ")"]
+  ],
+  "autoClosingPairs": [
+      ["{", "}"],
+      ["[", "]"],
+      ["(", ")"],
+      ["\"", "\""],
+      ["'", "'"]
```

Comment:
Are back slashes or single quotes used in the language? I haven't seen any examples using them.

Reply:
The back slashes are just to escape the `"` characters.

Single quote doesn't appear to be used. That auto close could be removed.